src/main/java/DatabaseInteractin/MainClass.java

```diff
@@ -0,0 +1,80 @@
+package DatabaseInteractin;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Scanner;
+
+public class MainClass {
+  public static void main(String[] args) throws SQLException {
+    Scanner sc = new Scanner(System.in);
```

Comment:
This looks duplicated

Reply:
This class will be removed later. It's used only to make tests on the database functions without gRPC. After running the tests and verifying that they are working, the methods are used in the gRPC implementation.

Reply:
Done.